src/test/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseServiceTest.java

```diff
@@ -359,4 +360,38 @@ public void saveResponseAcceptationShouldSucceedWhenPartAdmitPayImmediatelyWithN
         inOrder.verify(appInsights, once()).trackEvent(eq(CLAIMANT_RESPONSE_ACCEPTED),
             eq(REFERENCE_NUMBER), eq(claim.getReferenceNumber()));
     }
+
+    @Test
+    public void shouldNotRaiseGenericClaimantResponseEventOnSettlementAgreement() {
+        ClaimantResponse claimantResponse = SampleClaimantResponse
+            .ClaimantResponseAcceptation
+            .builder()
+            .withFormaliseOption(FormaliseOption.SETTLEMENT)
+            .withAmountPaid(new BigDecimal(100))
+            .build();
+
+        Claim claim = SampleClaim.builder()
+            .withResponseDeadline(LocalDate.now().minusMonths(2))
+            .withResponse(
+                SampleResponse.PartAdmission.builder().buildWithPaymentOptionBySpecifiedDate()
+            )
+            .withRespondedAt(LocalDateTime.now().minusDays(32))
+            .withClaimantResponse(claimantResponse)
+            .build();
+
+        when(claimService.getClaimByExternalId(eq(EXTERNAL_ID), eq(AUTHORISATION))).thenReturn(claim);
+        when(caseRepository.saveClaimantResponse(any(Claim.class), any(ResponseAcceptation.class), eq(AUTHORISATION)))
+            .thenReturn(claim);
+
+        InOrder inOrder = inOrder(caseRepository, formaliseResponseAcceptanceService, eventProducer, appInsights);
+
+        claimantResponseService.save(EXTERNAL_ID, claim.getSubmitterId(), claimantResponse, AUTHORISATION);
+
+        inOrder.verify(caseRepository, once()).saveClaimantResponse(any(Claim.class), eq(claimantResponse), any());
+        inOrder.verify(formaliseResponseAcceptanceService, once())
+            .formalise(any(Claim.class), any(ResponseAcceptation.class), eq(AUTHORISATION));
+        inOrder.verify(eventProducer, never()).createClaimantResponseEvent(any(Claim.class));
```

Comment:
`verify(eventProducer, never()).createClaimantResponseEvent(any(Claim.class));` can be on its own at the end. Doesn't need to be inOrder otherwise it gets muddled up with other events that must happen.